test/inputs/anonymous_functions.js

```diff
@@ -7,5 +7,9 @@ var testObj = {
 };
 
 testObj.somemethod('I shall not pass');
-testObj.gettext("I'm gonna get translated, yay!");
+testObj.pgettext("I'm gonna get translated, yay!");
 testObj.ngettext("I'm also gonna get translated!", "I'm the plural form!", 2);
+testObj.pgettext("context1", "I am translated in context!");
```

Comment:
I'm not sure this reflects the real-world context usage since I expect them to come from a variable usually. What do you think? (I know if they are coming from a variable it is impossible to extract them as we do here in a static analysis)